src/Maker/MakeRegistrationForm.php

```diff
@@ -276,14 +274,24 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'Controller\\'
         );
 
+        /*
+         * @legacy Conditional can be removed when MakerBundle no longer
+         *         supports Symfony < 6.0
```

Comment:
Couldn't we drop this once MakerBundle no longer supports 5.2? The thinking is: `UserPasswordHasherInterface` is new in 5.3. If an app is using 5.3, then we will ALWAYS give them the new class. So once MakerBundle requires 5.3 or higher, we can drop this. That's much sooner than 6.0 :)

Reply:
agreed - i was thinking when the old interface is removed, not when the new one was added.. lets do it!